docs/content/filtering.md

```diff
@@ -292,225 +424,255 @@ Prepare a file like this `filter-file.txt`
+    /home/user2/prefect  → remote:backup/user2/stuff
 
-You could of course choose `/` as the root too in which case your
-`files-from.txt` might look like this.
+Alternatively if `/` is chosen as root `files-from.txt` would be:
 
-    /home/user1/important
-    /home/user1/dir/file
-    /home/user2/stuff
+    /home/user1/42
+    /home/user1/dir/ford
+    /home/user2/prefect
 
-And you would transfer it like this
+The copy command would be:
 
     rclone copy --files-from files-from.txt / remote:backup
 
-In this case there will be an extra `home` directory on the remote:
+Then there will be an extra `home` directory on the remote:
 
-    /home/user1/important → remote:backup/home/user1/important
-    /home/user1/dir/file  → remote:backup/home/user1/dir/file
-    /home/user2/stuff     → remote:backup/home/user2/stuff
+    /home/user1/42       → remote:backup/home/user1/42
+    /home/user1/dir/ford → remote:backup/home/user1/dir/ford
+    /home/user2/prefect  → remote:backup/home/user2/prefect
 
 ### `--files-from-raw` - Read list of source-file names without any processing ###
-This option is same as `--files-from` with the only difference being that the input
-is read in a raw manner. This means that lines with leading/trailing whitespace and
-lines starting with `;` or `#` are read without any processing. [rclone lsf](/commands/rclone_lsf/)
-has a compatible format that can be used to export file lists from remotes, which
-can then be used as an input to `--files-from-raw`.
+
+This flag is the same as `--files-from` except that input is read in a
+raw manner. Lines with leading / trailing whitespace, and lines starting
+with `;` or `#` are read without any processing. [rclone lsf](/commands/rclone_lsf/) has
+a compatible format that can be used to export file lists from remotes for
+input to `--files-from-raw`. 
+
+### `--ignore-case` - make searches case insensitive ###
+
+By default rclone filter patterns are case sensitive. The `--ignore-case`
+flag makes all of the filters patterns on the command line case
+insensitive.
+
+Eg `--include "zaphod.txt"` does not match a file `Zaphod.txt`. With
+`--ignore-case` a match is made.
+
+## Quoting shell metacharacters ##
+
+Rclone commands with filter patterns containing shell metacharacters may
+not as work as expected in your shell and may require quoting.
+
+Eg linux, OSX (`*` metacharacter)
+
+  * `--include \*.jpg`
+  * `--include '*.jpg'`
+  * `--include='*.jpg'`
+
+Microsoft Windows expansion is done by the command not shell so `--include *.jpg` does
+not require quoting.
+
+If the rclone error
+`Command .... needs .... arguments maximum: you provided .... non flag arguments:`
+is encountered, the cause is commonly spaces within the name of a remote
+or flag. It can be corrected by quoting the values containing the spaces.
+
+Eg `rclone ls remote 1: --exclude /my documents/**` can be quoted
+`rclone ls 'remote 1:' --exclude '/my documents/**`.
```

Comment:
These should probably be "..." on Windows. I'd use '...' on unix.

Reply:
Yes, that seems a good approach to the single quotes, thanks.